spec/lookup_spec.rb

```diff
@@ -32,24 +30,28 @@
           :key_conditions=>{
             "component_key"=> {
               :comparison_operator=>"EQ",
-              :attribute_value_list=>[{"s"=>"id/9dd916afd5516828a91d259967fd394a"}]
+              :attribute_value_list=>[{"s"=>"id/218c835cec343537589dbf1619532e4d"}]
             },
             "batch_version"=>{
               :comparison_operator=>"EQ",
-              :attribute_value_list=>[{"n"=>"{:variant=>\"foo\"}"}]
+              :attribute_value_list=>[{"n"=>"0"}]
             }
           }
         }
       end
 
-      it "queries DynamoDb and returns a location" do
-        AWS::DynamoDB::Client::V20120810
-          .any_instance
-          .stub(:initialize)
+      it "queries DynamoDb and returns a location when not in cache" do
+        expect_any_instance_of(Dalli::ElastiCache).to receive(:initialize)
+        expect_any_instance_of(Dalli::ElastiCache).to receive(:client).and_return(Dalli::Client.new)
```

Comment:
@samfrench re: returning an actual object instance vs returning a double. I'd personally stick with returning an actual instance unless I needed to mock/stub any methods on the object. So keep it as is for now.